docs/runbooks/VirtControllerDown.md

```diff
@@ -0,0 +1,24 @@
+# VirtControllerDown
+
+## Meaning
+No running virt-controller was detected for the last 5 min. Virt-controller deployment has a default replica of 2 pods. 
+
+## Impact
+Complete failure in VM lifecycle management i.e. launching a new VM instance or shutting down an existing VM instance.
+
+## Diagnosis
+
+- Set the environment variable NAMESPACE
+
+    ```
+    export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
+    ```
+
+- Observe the status of the virt-controller deployment
+    ```
+    kubectl get deployment -n $NAMESPACE virt-controller -o yaml
+    ```
+Observe the logs of the kubecontroller manager pod, to see why it cannot create the virt-controller pods.
+
+## Mitigation
+Node resource exhaustion may be the root cause of this event.
```

Comment:
@iholder-redhat @rmohr  This doesn't seem to be a very helpful mitigation.
Can you please help with step for mitigation or how to open a case and what information to provide?

Reply:
@iholder-redhat @rmohr What should the user do in case they can't solve the issue?